recipe/meta.yaml

```diff
@@ -24,34 +24,32 @@ requirements:
     - python                                 # [build_platform != target_platform]
     - cross-python_{{ target_platform }}     # [build_platform != target_platform]
     - cython                                 # [build_platform != target_platform]
-    - numpy  >=1.21,<1.29                    # [build_platform != target_platform]
```

Comment:
The latest numpy version is 2.2.4. Should we go ahead and update to it if it looks good?

Reply:
I can run a quick test to see if this would be a simple upgrade to include in this PR.

Reply:
Holding off on numpy upgrade for now due to `catboost` compatibility issue https://github.com/catboost/catboost/issues/2671.